integration/docker/entrypoint.sh

```diff
@@ -67,18 +71,30 @@ done
 
 case ${service,,} in
   master)
-    bin/alluxio format
+    if [[ -n ${options} && ${options} != ${NO_FORMAT} ]]; then
+      echo 'invalid option ' ${options} ', expected ' ${NO_FORMAT}
+      exit 1
+    fi
+    if [[ ${options} != ${NO_FORMAT} ]]; then
+      bin/alluxio format
+    fi
     integration/docker/bin/alluxio-master.sh
     ;;
   worker)
-    bin/alluxio formatWorker
+    if [[ -n ${options} && ${options} != ${NO_FORMAT} ]]; then
```

Comment:
could we de-duplicate this check by checking before the case statement?

Reply:
I prefer in the case statement as the options validation is not applicable to all services (e.g. proxy), and we may have different options for master and worker

Reply:
sgtm